custom/conf/app.ini.sample

```diff
@@ -16,6 +16,8 @@ SCRIPT_TYPE = bash
 ANSI_CHARSET =
 ; Force every new repository to be private
 FORCE_PRIVATE = false
+; Default private when create a new repository, 0: last user repo visiblity, 1: private, 2: public
```

Comment:
Would it be better to use named constants ("last", "private", "public") e.g. like we do for `RUN_MODE`?